Makefile

```diff
@@ -162,7 +177,8 @@ release:
 	@sed -i -e 's/^VERSION := .*/VERSION := ${NEWVERSION}/' ./Makefile
 
 release-manifest:
-	@sed -i'' -e 's@image: $(REPOSITORY):.*@image: $(REPOSITORY):'"$(NEWVERSION)"'@' ./config/manager/manager.yaml ./deploy/gatekeeper.yaml
+		@sed -i'' -e 's@image: $(REPOSITORY):.*@image: $(REPOSITORY):'"$(NEWVERSION)"'@' ./config/manager/manager.yaml ./deploy/gatekeeper_kubebuilder_v2.yaml
+
```

Comment:
I don't think we need to update this since `gatekeeper_kubebuilder_v2.yaml` is only temporary. WDYT?

Reply:
I think we should keep things consistent and move them in lock step. This way the release process is: build/publish as normal, then rename YAML

Also, not renaming it risks modifying the original gatekeeper.yaml